### ISSUE TITLE
I created an add-on that uses Anki Persistence

### ISSUE THREAD

**Issue by hgiesel** [SimonLammer/anki-persistence#39]
Hi.
I created [this add-on](https://github.com/hgiesel/anki-set-randomizer), which utilizes Anki Persistence. I've given you credits on add-on page, and within the add-on itself.
If you like, I'd create a PR with a section "Projects which use Anki Persistence", or something like that, which would list my add-on.

**Comment by SimonLammer** [comment]
I'd much appreciate a PR of that sort.
I'll leave this issue open as a communication line. If you have any trouble using anki-persistence in your add-on or have ideas on how to update the readme, such that these troubles can be circumvented don't hesitate to notify me.
Good luck with your add-on.

**Comment by SimonLammer** [comment]
Since there don't seem to be any immediate questions, I'll close this.

**Comment by TowelSniffer** [comment]
I actually made a template that uses this. It isn't mine but I modified an existing template and it includes anki persistence. 

https://ankiweb.net/shared/info/1018121283

**Comment by hgiesel** [comment]
I also have some news on this: For quite some time, I've been working on a solution to bring a more versatile way of creating clozes, image occlusion, shuffling and more to Anki. In the course of this, I've written an Anki add-on called [Asset Manager](https://ankiweb.net/shared/info/656021484), which manages javascript in anki templates.

It also offers a hook for other add-ons to inject the Javascript into it. And so I've written an anki add-on, which injects Anki Persistence into this other add-on.

TLDR: I've written an add-on which helps users use Anki-Persistence.

What I'm asking for is a for a head ups, whether I can distribute it on AnkiWeb. It will still be 100% your javascript script, but it will be wrapped in an Anki add-on which is developed by me (you can see it in my fork of anki-persistence in the branch "assetmanager").

**Comment by SimonLammer** [comment]
Sure thing, go right ahead. I strongly encourage you to keep the comment denoting the anki-persistence version and link to the non-minified javascript in the card template.
